app/models/manageiq/providers/vmware/cloud_manager/vm/remote_console.rb

```diff
@@ -23,6 +23,7 @@ def remote_console_acquire_ticket_queue(protocol, userid)
       :class_name  => self.class.name,
       :instance_id => id,
       :method_name => 'remote_console_acquire_ticket',
+      :queue_name  => queue_name_for_ems_operations,
```

Comment:
The cloud_manager doesn't currently have an OperationsWorker so this will just go to a normal priority/generic worker but it is good practice to always set the queue_name here